src/spradd2/spraddsection.js

```diff
@@ -0,0 +1,63 @@
+import {
+  CreatorStorcard,
+  Desc,
+  DescPara,
+  EmtyDiv,
+  HeadingDiv,
+  LargeDiv,
+  LeftImageDiv,
+  LeftSideImage,
+  LeftsideDiv,
+  PinkH1,
+  PinkLineDiv,
+  RightSideDiveTOp,
+  RightsideDiv,
+  SmallSizeimage,
+  Subdiv,
+  TextDiv,
+  TextStyle,
+  WhiteCardDiv,
+} from "./spraddsection.styled";
+export function SpreadforCreator() {
+  return (
+    <>
+      <LargeDiv>
+        <RightSideDiveTOp>
+          <SmallSizeimage src="https://spreadd.io/assets/home/left-dot-patch.svg"></SmallSizeimage>
+        </RightSideDiveTOp>
+        <HeadingDiv>
+          <Subdiv>
+            <LeftsideDiv>
```

Comment:
please use proper name instead of left div, right dix, whitediv @Arbina001

Reply:
i have rename it